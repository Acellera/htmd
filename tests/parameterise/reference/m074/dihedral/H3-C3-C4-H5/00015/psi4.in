set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

O	 1.589393	 1.743790	 0.490967
C	 0.939996	 1.446181	 1.441141
C	 -0.368203	 0.694160	 1.410421
O	 1.295092	 1.742260	 2.685079
C	 -0.823008	 0.326187	 0.001381
H	 -0.248488	 -0.190941	 2.029274
H	 -1.108253	 1.309361	 1.914974
C	 -2.149452	 -0.435131	 -0.000098
H	 -0.054037	 -0.271430	 -0.477916
H	 -0.915791	 1.230380	 -0.592029
C	 -3.368779	 0.488503	 -0.000883
H	 -2.193746	 -1.087250	 -0.871175
H	 -2.195311	 -1.087803	 0.870418
C	 -3.346354	 1.505125	 -1.141597
H	 -3.422266	 1.013619	 0.950187
H	 -4.273627	 -0.111280	 -0.069069
H	 -2.469560	 2.143689	 -1.080920
H	 -4.223185	 2.144600	 -1.115337
H	 -3.327221	 1.009791	 -2.108339
C	 2.510153	 2.449014	 2.855607
H	 3.340132	 1.877578	 2.465577
H	 2.621562	 2.592138	 3.919082
H	 2.468423	 3.403855	 2.351579

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		15 11 8 12,
		1 2 3 6,
		9 5 3 6,
		1 2 4 20
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "23  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
