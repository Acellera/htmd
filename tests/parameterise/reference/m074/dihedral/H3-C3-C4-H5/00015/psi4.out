    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                              Psi4 (no tag) Driver

                          Git: Rev {detached?}  

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: Tue Jul  5 16:57:35 2016

    Process ID:  28786
    PSI4DATADIR: /export131/home/mjharvey/miniconda3/share
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

O	 1.589393	 1.743790	 0.490967
C	 0.939996	 1.446181	 1.441141
C	 -0.368203	 0.694160	 1.410421
O	 1.295092	 1.742260	 2.685079
C	 -0.823008	 0.326187	 0.001381
H	 -0.248488	 -0.190941	 2.029274
H	 -1.108253	 1.309361	 1.914974
C	 -2.149452	 -0.435131	 -0.000098
H	 -0.054037	 -0.271430	 -0.477916
H	 -0.915791	 1.230380	 -0.592029
C	 -3.368779	 0.488503	 -0.000883
H	 -2.193746	 -1.087250	 -0.871175
H	 -2.195311	 -1.087803	 0.870418
C	 -3.346354	 1.505125	 -1.141597
H	 -3.422266	 1.013619	 0.950187
H	 -4.273627	 -0.111280	 -0.069069
H	 -2.469560	 2.143689	 -1.080920
H	 -4.223185	 2.144600	 -1.115337
H	 -3.327221	 1.009791	 -2.108339
C	 2.510153	 2.449014	 2.855607
H	 3.340132	 1.877578	 2.465577
H	 2.621562	 2.592138	 3.919082
H	 2.468423	 3.403855	 2.351579

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		15 11 8 12,
		1 2 3 6,
		9 5 3 6,
		1 2 4 20
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "23  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
--------------------------------------------------------------------------

  Memory set to   2.000 GiB by Python script.

*** tstart() called on cx1-1-1-4.cx1.hpc.ic.ac.uk
*** at Tue Jul  5 16:57:35 2016


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,   2000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           O          1.968005504068     0.609786036924    -0.384076470443    15.994914619560
           C          1.318608504068     0.312177036924     0.566097529557    12.000000000000
           C          0.010409504068    -0.439843963076     0.535377529557    12.000000000000
           O          1.673704504068     0.608256036924     1.810035529557    15.994914619560
           C         -0.444395495932    -0.807816963076    -0.873662470443    12.000000000000
           H          0.130124504068    -1.324944963076     1.154230529557     1.007825032070
           H         -0.729640495932     0.175357036924     1.039930529557     1.007825032070
           C         -1.770839495932    -1.569134963076    -0.875141470443    12.000000000000
           H          0.324575504068    -1.405433963076    -1.352959470443     1.007825032070
           H         -0.537178495932     0.096376036924    -1.467072470443     1.007825032070
           C         -2.990166495932    -0.645500963076    -0.875926470443    12.000000000000
           H         -1.815133495932    -2.221253963076    -1.746218470443     1.007825032070
           H         -1.816698495932    -2.221806963076    -0.004625470443     1.007825032070
           C         -2.967741495932     0.371121036924    -2.016640470443    12.000000000000
           H         -3.043653495932    -0.120384963076     0.075143529557     1.007825032070
           H         -3.895014495932    -1.245283963076    -0.944112470443     1.007825032070
           H         -2.090947495932     1.009685036924    -1.955963470443     1.007825032070
           H         -3.844572495932     1.010596036924    -1.990380470443     1.007825032070
           H         -2.948608495932    -0.124212963076    -2.983382470443     1.007825032070
           C          2.888765504068     1.315010036924     1.980563529557    12.000000000000
           H          3.718744504068     0.743574036924     1.590533529557     1.007825032070
           H          3.000174504068     1.458134036924     3.044038529557     1.007825032070
           H          2.847035504068     2.269851036924     1.476535529557     1.007825032070

  Running in c1 symmetry.

  Rotational constants: A =      0.13415  B =      0.02022  C =      0.01985 [cm^-1]
  Rotational constants: A =   4021.67016  B =    606.13476  C =    594.94814 [MHz]
  Nuclear repulsion =  464.414269162866560

  Charge       = 0
  Multiplicity = 1
  Electrons    = 72
  Nalpha       = 36
  Nbeta        = 36

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Number of shells: 82
    Number of basis function: 163
    Number of Cartesian functions: 163
    Spherical Harmonics?: false
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A        163     163       0       0       0       0
   -------------------------------------------------------
    Total     163     163      36      36      36       0
   -------------------------------------------------------

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):              1430
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 342
    Number of basis function: 1079
    Number of Cartesian functions: 1079
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 2.6637628325E-03.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:  -265.57199916555709   -2.65572e+02   7.02840e-02 
   @DF-RHF iter   2:   -55.04784127392173    2.10524e+02   7.97471e-02 DIIS
   @DF-RHF iter   3:  -223.93551137036064   -1.68888e+02   6.47700e-02 DIIS
   @DF-RHF iter   4:  -300.30482605469507   -7.63693e+01   6.07450e-02 DIIS
   @DF-RHF iter   5:  -387.99997343566923   -8.76951e+01   3.97317e-02 DIIS
   @DF-RHF iter   6:  -421.97939577660469   -3.39794e+01   8.54642e-03 DIIS
   @DF-RHF iter   7:  -422.76796376074543   -7.88568e-01   4.41312e-03 DIIS
   @DF-RHF iter   8:  -422.94798731406587   -1.80024e-01   1.06157e-03 DIIS
   @DF-RHF iter   9:  -422.96569347663461   -1.77062e-02   3.90441e-04 DIIS
   @DF-RHF iter  10:  -422.96802921293551   -2.33574e-03   1.28524e-04 DIIS
   @DF-RHF iter  11:  -422.96834582803518   -3.16615e-04   4.77751e-05 DIIS
   @DF-RHF iter  12:  -422.96839626944626   -5.04414e-05   1.32887e-05 DIIS
   @DF-RHF iter  13:  -422.96840182384034   -5.55439e-06   4.23659e-06 DIIS
   @DF-RHF iter  14:  -422.96840235817064   -5.34330e-07   1.47434e-06 DIIS
   @DF-RHF iter  15:  -422.96840242238414   -6.42135e-08   5.59865e-07 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A    -20.603526     2A    -20.545362     3A    -11.372006  
       4A    -11.280787     5A    -11.235175     6A    -11.217920  
       7A    -11.217023     8A    -11.216278     9A    -11.208934  
      10A     -1.469220    11A     -1.362654    12A     -1.096389  
      13A     -1.040997    14A     -0.984091    15A     -0.931807  
      16A     -0.861413    17A     -0.782334    18A     -0.761401  
      19A     -0.701452    20A     -0.683151    21A     -0.672012  
      22A     -0.644526    23A     -0.621148    24A     -0.593288  
      25A     -0.587007    26A     -0.564451    27A     -0.536378  
      28A     -0.534120    29A     -0.521629    30A     -0.504443  
      31A     -0.496311    32A     -0.466711    33A     -0.458813  
      34A     -0.458054    35A     -0.452304    36A     -0.436531  

    Virtual:                                                              

      37A      0.197900    38A      0.222511    39A      0.258294  
      40A      0.265501    41A      0.267389    42A      0.279651  
      43A      0.299640    44A      0.310154    45A      0.313635  
      46A      0.322087    47A      0.326872    48A      0.332437  
      49A      0.349231    50A      0.356686    51A      0.377756  
      52A      0.393857    53A      0.397993    54A      0.414796  
      55A      0.443569    56A      0.468058    57A      0.491869  
      58A      0.530464    59A      0.574687    60A      0.701618  
      61A      0.720284    62A      0.727345    63A      0.739131  
      64A      0.750346    65A      0.772402    66A      0.782808  
      67A      0.792735    68A      0.804177    69A      0.843251  
      70A      0.861298    71A      0.906668    72A      0.931665  
      73A      0.950004    74A      0.974875    75A      0.990460  
      76A      0.997139    77A      1.003759    78A      1.034269  
      79A      1.085759    80A      1.113721    81A      1.122871  
      82A      1.127013    83A      1.135950    84A      1.149434  
      85A      1.155498    86A      1.163453    87A      1.179346  
      88A      1.185807    89A      1.197581    90A      1.206283  
      91A      1.216263    92A      1.231404    93A      1.256434  
      94A      1.257275    95A      1.260971    96A      1.283814  
      97A      1.287645    98A      1.332289    99A      1.345714  
     100A      1.382072   101A      1.419658   102A      1.453359  
     103A      1.558606   104A      1.646430   105A      1.664860  
     106A      1.674873   107A      1.711774   108A      1.766893  
     109A      1.799143   110A      1.846484   111A      1.872484  
     112A      1.896242   113A      1.916720   114A      1.990370  
     115A      2.021932   116A      2.037393   117A      2.077058  
     118A      2.121907   119A      2.155408   120A      2.164566  
     121A      2.191301   122A      2.214408   123A      2.271425  
     124A      2.283508   125A      2.287971   126A      2.325296  
     127A      2.356816   128A      2.376772   129A      2.405819  
     130A      2.416457   131A      2.440430   132A      2.462295  
     133A      2.484559   134A      2.520045   135A      2.594919  
     136A      2.634400   137A      2.670578   138A      2.684034  
     139A      2.689799   140A      2.713080   141A      2.739831  
     142A      2.812462   143A      2.827835   144A      2.844515  
     145A      2.910251   146A      2.943626   147A      2.982052  
     148A      3.049492   149A      3.067712   150A      3.117874  
     151A      3.162621   152A      3.350356   153A      3.451611  
     154A      3.580137   155A      4.415290   156A      4.549763  
     157A      4.598238   158A      4.643969   159A      4.716580  
     160A      4.788247   161A      4.865180   162A      4.940225  
     163A      5.020369  

    Final Occupation by Irrep:
              A 
    DOCC [    36 ]

  Energy converged.

  @DF-RHF Final Energy:  -422.96840242238414

   => Energetics <=

    Nuclear Repulsion Energy =            464.4142691628665602
    One-Electron Energy =               -1496.5262201503473989
    Two-Electron Energy =                 609.1435485650952160
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -422.9684024223856795

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:   -10.0142      Y:    -1.7751      Z:    -3.8033

  Electronic Dipole Moment: (a.u.)
     X:     9.7527      Y:     1.6835      Z:     4.4515

  Dipole Moment: (a.u.)
     X:    -0.2615      Y:    -0.0916      Z:     0.6482     Total:     0.7049

  Dipole Moment: (Debye)
     X:    -0.6646      Y:    -0.2327      Z:     1.6476     Total:     1.7918


  Saving occupied orbitals to File 180.

*** tstop() called on cx1-1-1-4.cx1.hpc.ic.ac.uk at Tue Jul  5 16:58:00 2016
Module time:
	user time   =      24.14 seconds =       0.40 minutes
	system time =       0.24 seconds =       0.00 minutes
	total time  =         25 seconds =       0.42 minutes
Total time:
	user time   =      24.14 seconds =       0.40 minutes
	system time =       0.24 seconds =       0.00 minutes
	total time  =         25 seconds =       0.42 minutes


Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the  density matrix

  Nuclear Dipole Moment: (a.u.)
     X:   -10.0142      Y:    -1.7751      Z:    -3.8033

  Electronic Dipole Moment: (a.u.)
     X:     9.7527      Y:     1.6835      Z:     4.4515

  Dipole Moment: (a.u.)
     X:    -0.2615      Y:    -0.0916      Z:     0.6482     Total:     0.7049

  Dipole Moment: (Debye)
     X:    -0.6646      Y:    -0.2327      Z:     1.6476     Total:     1.7918

  Quadrupole Moment: (Debye Ang)
    XX:   -55.1069     YY:   -54.7538     ZZ:   -57.1532
    XY:     1.2042     XZ:     6.1730     YZ:     2.8122

  Traceless Quadrupole Moment: (Debye Ang)
    XX:     0.5644     YY:     0.9175     ZZ:    -1.4819
    XY:     1.2042     XZ:     6.1730     YZ:     2.8122

  Mulliken Charges: (a.u.)
   Center  Symbol    Alpha    Beta     Spin     Total
       1     O     4.28589  4.28589  0.00000 -0.57179
       2     C     2.60551  2.60551  0.00000  0.78897
       3     C     3.20847  3.20847  0.00000 -0.41695
       4     O     4.30720  4.30720  0.00000 -0.61439
       5     C     3.15903  3.15903  0.00000 -0.31807
       6     H     0.40030  0.40030  0.00000  0.19940
       7     H     0.40051  0.40051  0.00000  0.19898
       8     C     3.15903  3.15903  0.00000 -0.31807
       9     H     0.40933  0.40933  0.00000  0.18134
      10     H     0.40791  0.40791  0.00000  0.18418
      11     C     3.15367  3.15367  0.00000 -0.30735
      12     H     0.41817  0.41817  0.00000  0.16366
      13     H     0.42189  0.42189  0.00000  0.15622
      14     C     3.24374  3.24374  0.00000 -0.48747
      15     H     0.42430  0.42430  0.00000  0.15139
      16     H     0.41894  0.41894  0.00000  0.16213
      17     H     0.41835  0.41835  0.00000  0.16331
      18     H     0.41822  0.41822  0.00000  0.16356
      19     H     0.42041  0.42041  0.00000  0.15917
      20     C     3.09243  3.09243  0.00000 -0.18486
      21     H     0.40903  0.40903  0.00000  0.18193
      22     H     0.40847  0.40847  0.00000  0.18306
      23     H     0.40918  0.40918  0.00000  0.18164

   Total alpha = 36.00000, Total beta = 36.00000, Total charge =  0.00000


*** PSI4 exiting successfully. Buy a developer a beer!
